kadi/commands/tests/test_commands.py

```diff
@@ -502,6 +531,24 @@ def test_cmds_scenario(stop_date_2020_12_03):
 def test_command_set_bsh():
     cmds = get_cmds_from_event("2000:001", "Bright star hold", "")
     exp = [
+        "2000:001:00:00:00.000 | COMMAND_SW       | CODISASX   | CMD_EVT  | "
+        "event=Bright_star_hold, event_date=2000:001:00:00:00, msid=CODISASX, "
+        "codisas1=12",
```

Comment:
These look a little odd (and don't fully confirm the right behavior) due to truncation, though that's more of a problem with pformat_like_backstop than this PR.

Reply:
I think technically this test should just be run using max_params_width=None for pformat_like_backstop.

Reply:
Fixed.